src/pages/search.jsx

```diff
@@ -16,6 +16,7 @@ import {
 const propTypes = {
   isSearching: PropTypes.bool,
   lastPage: PropTypes.string,
+  resetSearch: PropTypes.func,
```

Comment:
`PropTypes.func.isRequired`

If the component try's to unMount and this function is not available, the app will crash

Reply:
Ok, I modified it.